frameworks/FSharp/giraffe/benchmark_config.json

```diff
@@ -5,19 +5,20 @@
       "default": {
         "plaintext_url": "/plaintext",
         "json_url": "/json",
+        "fortune_url": "/fortunes",
         "port": 8080,
         "approach": "Realistic",
         "classification": "fullstack",
-        "database": "None",
+        "database": "Postgres",
         "framework": "giraffe",
         "language": "F#",
-        "orm": "Raw",
+        "orm": "micro",
         "platform": ".NET",
         "flavor": "CoreCLR",
         "webserver": "Kestrel",
         "os": "Linux",
         "database_os": "Linux",
-        "display_name": "Giraffe",
+        "display_name": "Giraffe, Dapper",
```

Comment:
Not a big fan of this, but since we don't have a key for the name of your database approach/library, I guess it will do.